sfcgal/lwgeom_sfcgal.c

```diff
@@ -375,6 +375,30 @@ Datum sfcgal_straight_skeleton(PG_FUNCTION_ARGS)
 	PG_RETURN_POINTER(output);
 }
 
+PG_FUNCTION_INFO_V1(sfcgal_straight_skeleton_distance_in_m);
+Datum sfcgal_straight_skeleton_distance_in_m(PG_FUNCTION_ARGS)
+{
```

Comment:
You should add an ifdef SFCGAL >= 1.3.8

see https://git.osgeo.org/gitea/postgis/postgis/pulls/98/files for example.